radio/src/opentx.cpp

```diff
@@ -2572,7 +2572,7 @@ int main()
   // lcdSetRefVolt(25);
 #endif
 
-#if defined(PCBTARANIS) || defined(PCBHORUS)
+#if defined(SPLASH) && (defined(PCBTARANIS) || defined(PCBHORUS))
```

Comment:
why? I thought I disabled the possibility of removing the SPLASH on those targets because the initialization was long (the audio task searches for a lot of files), I may be wrong though it's a long time I did it